src/open_r1/configs.py

```diff
@@ -31,7 +31,8 @@ class GRPOConfig(trl.GRPOConfig):
         metadata={"help": ("The benchmarks to run after training.")},
     )
     callbacks: list[str] = field(
-        default_factory=lambda: [], metadata={"help": ("The callbacks to run during training.")}
+        default_factory=lambda: [],
```

Comment:
strange reformatting, it should fit in 119 characters

Reply:
yes just checking the arguments...